app/controllers/task_controller.rb

```diff
@@ -6,26 +6,50 @@ def index
   end
```

Comment:
Per Rails naming conventions, this controller name should be plural: TasksController